node/src/neighborhood/mod.rs

```diff
@@ -106,6 +108,9 @@ pub struct Neighborhood {
     db_password_opt: Option<String>,
     logger: Logger,
     tools: NeighborhoodTools,
+    exit_locations_opt: Option<HashMap<usize, Vec<String>>>,
```

Comment:
I may come back and change this comment, but at the moment I think this could probably be eliminated, because it doesn't need to survive the processing of the set-exit-locations message from the UI.